test/msg_cache_unittest.cpp

```diff
@@ -135,7 +135,7 @@ TEST(Cache, easySurroundingInterval)
 }
 
 
-std::shared_ptr<Msg const> buildMsg(double time, int data)
+std::shared_ptr<Msg const> buildMsg(int32_t time, int data)
```

Comment:
Should the argument be named `seconds`?

Reply:
Yes, also is there a reason that we don't have a `double` constructor for time in ROS2?

Reply:
> is there a reason that we don't have a `double` constructor for time in ROS2?

Probably just not implemented yet.